### PR TITLE
Switch to npm

### PR DESCRIPTION
**Pull request by ferferga**
Changes this project package management from ``yarn`` to ``npm``, as it's being done in all the organization repositories.

Major reasons for switching:

* All the yarn's advantages over npm are non-existent today
* Yarn introduced some breaking changes in 2.x that caused controversies and avoid us to upgrade directly, which means we are stuck in a version in maintenance mode. So far, all the npm major updates could be updated like a charm, and it's likely they will commit to it. The same can't be said about yarn.
* Npm comes with the node installation, so it's frictionless for the user
* Npm is more mainstream and all the package maintainers add instructions with how to use it to install their packages. The same doesn't apply to yarn. Although adapting the commands to yarn is trivial, for new contributors it might cause some friction.

## Extra

* Added dependabot for dependency updates in GitHub Actions

### CODE REVIEW

File: src/components/commandHandler.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ import { playbackManager } from './playbackManager';
 import { DocumentManager } from './documentManager';
 
 export abstract class CommandHandler {
-    private static playerManager: cast.framework.PlayerManager;
+    private static playerManager: framework.PlayerManager;
```
Comment (hawken93) [inline src/components/commandHandler.ts:27]: just a question here, I don't fully understand why "cast" is implicit here..

**Review by hawken93 — APPROVED** [review]
I didn't see anything that I wanted to do differently. LGTM :)

File: src/components/commandHandler.ts (hunk 1 above)
Comment (ferferga) [reply src/components/commandHandler.ts:27]: Found [this](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/28190#issuecomment-414637374) issue, basically the typings are not exported in the index, but in framework.

Shouldn't affect runtime though as it's all TypeScript abstraction as usual :)

Good catch!

**Comment by ferferga** [comment]
I was able to test this today on an ATV device and I had no issues with the switch, so I'm merging this :)

**Pull request merged**